Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -1614,12 +1784,35 @@ def fetch_incidents(client, first_fetch_time, last_run: dict = None):
 
     for raw_incident in raw_incidents:
         incident_id = raw_incident.get('incident_id')
+        incident_data = get_incident_extra_data_command(client, {"incident_id": incident_id,
+                                                                 "alerts_limit": 1000})[2].get('incident')
+
+        sort_all_lists_incident_fields(incident_data)
+
         description = raw_incident.get('description')
         occurred = timestamp_to_datestring(raw_incident['creation_time'], TIME_FORMAT + 'Z')
         incident = {
             'name': f'#{incident_id} - {description}',
             'occurred': occurred,
-            'rawJSON': json.dumps(raw_incident)
+            'rawJSON': json.dumps(incident_data),
+            'xdrincidentid': incident_id,
+            'xdrdetectiontime': incident_data.get('detection_time'),
+            'xdrstatus': incident_data.get('status'),
+            'xdrdescription': description,
+            'xdrassigneduseremail': incident_data.get('assigned_user_mail'),
+            'xdrassigneduserprettyname': incident_data.get('assigned_user_pretty_name'),
+            'xdralertcount': incident_data.get('alert_count'),
+            'xdrlowseverityalertcount': incident_data.get('low_severity_alert_count'),
+            'xdrmediumseverityalertcount': incident_data.get('med_severity_alert_count'),
+            'xdrhighseverityalertcount': incident_data.get('high_severity_alert_count'),
+            'xdrusercount': incident_data.get('user_count'),
+            'xdrhostcount': incident_data.get('host_count'),
+            'xdrnotes': incident_data.get('notes'),
+            'xdrurl': incident_data.get('xdr_url'),
+            'xdralerts': incident_data.get('alerts'),
+            'xdrresolvecomment': incident_data.get('resolve_comment'),
+            'xdrnetworkartifacts': incident_data.get('network_artifacts'),
+            'xdrfileartifacts': incident_data.get('file_artifacts'),
```

Comment:
don't we want all of those in the mapping?

Reply:
correct. it should be in the mapping